src/main/java/uk/gov/ons/ssdc/caseprocessor/schedule/CaseClassifier.java

```diff
@@ -1,21 +1,41 @@
 package uk.gov.ons.ssdc.caseprocessor.schedule;
 
+import java.util.Set;
 import java.util.UUID;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
+import uk.gov.ons.ssdc.caseprocessor.rasrm.service.RasRmSampleSummaryCollexLinkService;
 import uk.gov.ons.ssdc.common.model.entity.ActionRule;
 
 @Component
 public class CaseClassifier {
+  private static final Set<String> MANDATORY_COLLEX_METADATA =
+      Set.of("rasRmSampleSummaryId", "rasRmCollectionExerciseId");
 
   private final JdbcTemplate jdbcTemplate;
+  private final RasRmSampleSummaryCollexLinkService rasRmSampleSummaryCollexLinkService;
 
-  public CaseClassifier(JdbcTemplate jdbcTemplate) {
+  public CaseClassifier(
+      JdbcTemplate jdbcTemplate,
+      RasRmSampleSummaryCollexLinkService rasRmSampleSummaryCollexLinkService) {
     this.jdbcTemplate = jdbcTemplate;
+    this.rasRmSampleSummaryCollexLinkService = rasRmSampleSummaryCollexLinkService;
   }
 
   public void enqueueCasesForActionRule(ActionRule actionRule) {
+    if (actionRule
+        .getCollectionExercise()
+        .getSurvey()
+        .getSampleDefinitionUrl()
+        .endsWith("business.json")) {
```

Comment:
Type on survey would be nicer

Reply:
I think "inspired by" would be misleading, but this technique is being used in the INT team: https://github.com/ONSdigital/sdc-int-rh-service/blob/e8d8b09961e3273c522c2a9bfd4d58d01d5aff26/src/main/java/uk/gov/ons/ctp/integration/rhsvc/event/impl/CaseEventReceiverImpl.java#L55

"Survey type" is contentious, because it would require code changes every time there's a new "type" which we are going to great lengths to avoid.

The business integration remains tactical at this stage, so I think the check is adequate to keep the RAS-RM code separate from the rest.